Code/Engine/Core/World/Implementation/World.cpp

```diff
@@ -632,7 +632,7 @@ ezWorldModule* ezWorld::GetModule(const ezRTTI* pRtti)
   return nullptr;
 }
 
-const ezWorldModule* ezWorld::GetModuleReadOnly(const ezRTTI* pRtti) const
+const ezWorldModule* ezWorld::GetModule(const ezRTTI* pRtti) const
```

Comment:
There are both variants now. I've re-added the GetModule const method for consistency, since we don't use the ReadOnly suffix elsewhere. (we would need to add it to all const methods of ezWorld and ezGameObject, potentially ezComponent etc)